guides/source/action_cable_overview.md

```diff
@@ -780,6 +780,15 @@ The PostgreSQL adapter uses Active Record's connection pool, and thus the
 application's `config/database.yml` database configuration, for its connection.
 This may change in the future. [#27214](https://github.com/rails/rails/issues/27214)
 
+The adapter uses the [PostgreSQL's NOTIFY command](https://www.postgresql.org/docs/16/sql-notify.html), which enforces a fixed payload limit of 8000 bytes. To address this limitation, you can enable the `compress_payloads` option for payload compression, reducing the data size to be broadcast.
```

Comment:
I wouldn't phrase the compression option as "adressing this limitation". In practice it makes the limit ~3x bigger but doesn't prevent hitting it, which could be more directly communicated.

Reply:
True, I've modified this, what do you think now?

Reply:
Better.

I think it makes sense to shoot for a generic compression option for all ActionCable adapters though, at which point the documentation should probably be revisited.